src/svg-to-vectordrawable.js

```diff
@@ -609,6 +615,9 @@ JS2XML.prototype.refactorData = function(data, floatPrecision, fillBlack, tint)
                     });
                 }
             }
+
+            // Remove original gradient element
+            gradient.parentNode.spliceContent(gradient.parentNode.children.indexOf(gradient), 1, []);
```

Comment:
Here we remove the original gradient element after it has been copied over to all elements referencing it.